data/metadata.json

```diff
@@ -0,0 +1,5823 @@
+{
+  "metaData": [
+    {
+      "path": "$.document",
+      "user_documentation": {
+        "specification": "docs/user/document-spec.en.md",
+        "usage": "docs/user/document-usage.en.md"
+      },
+      "label": "DocumentLabel",
+      "description": "DocumentDescription",
+      "type": "OBJECT",
+      "mandatory": {
+        "csaf_base": "mandatory",
+        "csaf_security_incident_response": "mandatory",
+        "csaf_informational_advisory": "mandatory",
+        "csaf_security_advisory": "mandatory",
+        "csaf_vex": "mandatory"
+      }
```

Comment:
These values are parsed from the spreadsheet we defined on 30.03.22. I will update them with the latest data once the spreadsheet was fully filled and reviewed.

Reply:
updated with the current state (spreadsheet from 12.04.22)